tests/time_tests/c++/pipeline.h

```diff
@@ -0,0 +1,19 @@
+#include <chrono>
+#include <functional>
+
+#include <inference_engine.hpp>
+using namespace InferenceEngine;
+
+
+int runPipeline(float &duration) {
+    duration = []() {
+        auto t = std::chrono::high_resolution_clock::now();
+
+        Core ie;
```

Comment:
think about handling exceptions in `runPipeline` (it's required if IPC will be implemented)

Reply:
Done